src/rocksdb.rs

```diff
@@ -2095,11 +2096,27 @@ impl Env {
     pub fn new_mem() -> Env {
         unsafe {
             Env {
-                inner: crocksdb_ffi::crocksdb_create_mem_env(),
+                inner: crocksdb_ffi::crocksdb_mem_env_create(),
             }
         }
     }
 
+    // Create a ctr encrypted env with the default env and a given ciper text.
+    // The length of ciper text must equal to the block size.
+    // The block size must be 2^n, and must be less or equal to 2048.
+    // The recommanded block size are 1024, 512 and 256.
+    pub fn new_default_ctr_encrypted_env(block_size: usize, ciphertext: &[u8]) -> Env {
```

Comment:
Since it requires the `block_size` to be the same as the length of `ciphertext` and needs to assert this in different places, should we just leave out the `block_size` argument and infer it from the length of `ciphertext`?